crate2nix/templates/build.nix.tera

```diff
@@ -168,7 +168,7 @@ rec {
             usesDefaultFeatures = false;
             {%- endif -%}
             {%- if dependency.target %}
-            target = {{dependency.target | safe | cfg_to_nix_expr}};
+            target = {{dependency.target | safe | cfg_to_nix_expr | safe }};
```

Comment:
I don't think that the first safe is still needed, this should be sufficient:

```
            target = {{dependency.target | cfg_to_nix_expr | safe }};
```

Am I missing something?